docs/hub/datasets-duckdb.md

```diff
@@ -1,43 +1,72 @@
 # DuckDB
 
 [DuckDB](https://github.com/duckdb/duckdb) is an in-process SQL [OLAP](https://en.wikipedia.org/wiki/Online_analytical_processing) database management system.
-Since it supports [fsspec](https://filesystem-spec.readthedocs.io) to read and write remote data, you can use the Hugging Face paths ([`hf://`](https://huggingface.co/docs/huggingface_hub/guides/hf_file_system#integrations)) to read and write data on the Hub:
+You can use the Hugging Face paths (`hf://`) to access data on the Hub:
 
-First you need to [Login with your Hugging Face account](../huggingface_hub/quick-start#login), for example using:
+The [DuckDB CLI](https://duckdb.org/docs/api/cli/overview.html) (Command Line Interface) is a single, dependency-free executable. 
```

Comment:
maybe also mention that there are APIs for many different programming languages, including python
(i assume here that most of our audience are python developers)
and that all the code in these sections can be run in python wrapped within `duckdb.sql()` for example.

Reply:
Done, thanks!